lib/hades/mentorships/request.ex

```diff
@@ -0,0 +1,56 @@
+defmodule Hades.Mentorships.Request do
+  use Ecto.Schema
+
+  import Ecto.Changeset
+  import Hades.Helpers.Enum
+
+  alias Hades.Mentorships.Request
+
+  enum "status" do
```

Comment:
Why are you converting all those atoms to integers? By doing that we always need to convert them back to atom to compare the params, like you did above on `request_skill_area` and `request_mentoree_level`. If we save it as strings we don't need to do this kind of thing `skill_area == Request.skill_area[:fullstack]`.